src/components/Pupils/QuizRenderer/QuizRender.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC } from "react";
+import {
+  OakFlex,
+  OakHeading,
+  OakPrimaryButton,
+  OakTypography,
+  OakRadioGroup,
+  OakRadioButton,
+} from "@oak-academy/oak-components";
+
+import useQuizEngineContext from "../QuizEngineProvider/useQuizEngineContext";
+
+import { QuestionStem } from "@/components/QuizContainerNew/QuestionsListNew/QuestionListItemNew/QuestionStem";
+
+export const QuizRenderer: FC = () => {
+  const quizContext = useQuizEngineContext();
+
+  const {
+    currentQuestionData,
+    currentQuestionIndex,
+    questionState,
+    handleSubmitMCAnswer,
+    handleNextQuestion,
+  } = quizContext;
+
+  const { questionStem, answers, questionUid } = currentQuestionData || {};
```

Comment:
This isn't a great way to handle there being no data. We should think what we want to happen in this scenario.